test/mpi/maint/dtp-test-config.txt

```diff
@@ -13,45 +13,40 @@
 # NOTE: the second (ssv args), fourth (timeLimit) and seventh (maxtestsize) fields are optional.
 
 attr/fkeyvaltype::1::1:1024:
-coll/bcast::1 512 262144:timeLimit=420:4:16:1024
-coll/bcast_comm_world_only::1 512 262144:timeLimit=360:10:16:1024
+coll/bcast::1 512::4:16:1024
```

Comment:
Should the comment above be amended to describe the `mem=` argument?

Reply:
Sure.